src/v0/sources/shopify/util.js

```diff
@@ -176,6 +213,7 @@ const checkAndUpdateCartItems = async (inputEvent, metricMetadata) => {
       });
     }
   } catch (e) {
+    logger.debug(`{{SHOPIFY::}} itemsHash get call Failed due redis error ${e}`);
```

Comment:
`logger.debug(`{{SHOPIFY::}} Failed to retrieve itemsHash from Redis due to an error: ${e}`);`